Documentation/ChangeLog/Index.rst

```diff
@@ -5,13 +5,25 @@
 Change Log
 ==========
 
-Version 8.10.0
+Version 8.10.2
 --------------
 
 todo: tell about the important stuff here in verbose text
```

Comment:
we should either remove this line, or put some useful information here. That can be used as well for the upload comment in TER by the way.